templates/web/fixmystreet.com/about/press

```diff
@@ -0,0 +1,46 @@
+[% INCLUDE 'header.html',
+    title = loc('Press'),
+    bodyclass = 'twothirdswidthpage' %]
+
+[% INCLUDE 'about/_sidebar.html' %]
+
+<h1>Press resources</h1>
+
+<h2>About FixMyStreet</h2>
+
+<p>
+    FixMyStreet is an independent web service run by the charity <a href="https://www.mysociety.org/">mySociety</a> to make it easier for anyone to report a problem in their local area without needing to have any knowledge of council boundaries or responsibilities.
+</p>
+
+<p>
+You can find the answers to many of our most frequently asked questions on <a href="https://www.fixmystreet.com/faq">our FAQ page</a>.
```

Comment:
Doesn't matter, but to note for internal links on the same domain, you don't need to specify the domain, so this link could also be "/faq" (and then would work on developers' local machines too, which have a different domain).